converter/converter/cisu_converter.py

```diff
@@ -34,62 +34,78 @@ class CISUConverterV3:
         "medicalNote",
         "decision",
         "perimeter",
-        "interventionType", 
+        "interventionType",
         "qualification.origin",
         "qualification.details",
         "location.detailedAddress.highway",
         "location.geometry.point.isAml"
     ]
 
+    CISU_PATHS_TO_ADD_TO_INITIAL_ALERT_NOTES =[
+        '$.qualification.victims',
+        '$.initialAlert.attachment',
+        '$.initialAlert.callTaker',
+        '$.freetext',
+        '$.newAlert'
+    ]
+
     @classmethod
     def from_cisu(cls, input_json: Dict[str, Any]) -> Dict[str, Any]:
         """
         Convert from CISU to Health format
-        
+
         Args:
             input_json: Input EDXL CISU JSON
-            
+
         Returns:
             Converted EDXL Health JSON
         """
+        def add_location_detail(json_data: Dict[str,Any]):
+            if is_field_completed(json_data,'$.location.city.detail'):
+                if not is_field_completed(json_data, '$.location.freetext'):
+                    json_data['location']['freetext']=''
+                json_data['location']['freetext']+= " Détails de commune : " + json_data['location']['city']['detail']
```

Comment:
```suggestion
                json_data['location']['freetext']+= "\nDétails de commune : " + json_data['location']['city']['detail']
```
Ca rajoute un saut à la ligne même s'il n'y a rien dans freetext mais je pense qu'en terme de séparation c'est plus visuel pour les opérateurs et que ça évite que ça se mélange bizarrement avec la fin du freetext précédent, non ?